### ISSUE TITLE
Problems finding k8s schema file

### ISSUE THREAD

**Issue by boskisam** [datreeio/helm-datree#20]
**Describe the bug**
Wanted to give datree a try, installed helm plugin to check on my charts. Yaml validation is OK, Kubernetes schema validation cannot find file. Tried with example repo, no luck.

**To Reproduce**
1. `helm plugin install https://github.com/datreeio/helm-datree`
2. `git clone https://github.com/datreeio/examples.git`
3. `helm datree test examples/helm-chart/nginx`

**Expected behavior**
K8s schema is validated, starting policy check.

**Screenshots**
![image](https://user-images.githubusercontent.com/14150150/144378674-92253a44-7d29-4588-8352-f5c91bebeae3.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - kernel: 5.11.0-41-generic
 - helm: version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"} (installed by snap)

**Datree plugin version (run `helm datree verion`):**
 - Version: 0.14.49

**Additional context**  
Same happens with passing values
`helm datree test helm-charts/app/  -- --values compiled/DEV1/php.yml`

**Comment by eyarz** [comment]
another good way to debug if it's permission or a plugin issue will be to run the following command:
`helm template helm-chart/nginx | datree test -`
if you will see that same error ("empty file") - it's a permission issue.

**Comment by eyarz** [comment]
@boskisam do you have any update on that?

**Comment by boskisam** [comment]
Hi, 
sorry for my negligence. 
The file is created, permissions seem correct, but it is empty.
![image](https://user-images.githubusercontent.com/14150150/145274121-fd011472-04da-4391-a16e-56f83c9bb28f.png)
also tried with running helm as sudo. No luck either...

Since you brought up permissions, the problem might be with using the /tmp directory for datree. Some part of the **plugin** might use the system-wide temp, and **helm** can only use its' own _snap temp_ folder.

**Comment by boskisam** [comment]
I think i've foud a problem (or at least something that could help resolving the issue).
I've downloaded the datree binary, and ran `helm template ./helm-charts/app | datree test -` in my project folder.
It seems to me, that datree is trying to create the **/tmp** path based on a relative path to current directory. 
`$ helm template ./helm-charts/app | datree test -
>>  File: ../../../../../tmp/datree_temp_3723076163.yaml`

If i run the helm plugin command it resolves to absolute path
`$ helm template ./helm-charts/app | datree test -
>>  File: ../../../../../tmp/datree_temp_3723076163.yaml`

I did not review helm plugin code, but assuming it does `helm template xxx > $file && datree test $file` - the seems to be helm seeing only its' own /tmp dir, and datree only seeing system /tmp.
